src/propTypes.js

```diff
@@ -256,6 +257,17 @@ export const phenopacketPropTypesShape = PropTypes.shape({
     updated: PropTypes.string,  // ISO datetime string
 });
 
+export const experimentPropTypesShape = PropTypes.shape({
+    id: PropTypes.string.isRequired,
+    // subject: PropTypes.oneOfType([individualPropTypesShape, PropTypes.string]).isRequired,
```

Comment:
Ideally this can be filled in from what we already know about the endpoint

Reply:
agreed

Reply:
I will update this after the first test on signature